src/components/NavBar.jsx

```diff
@@ -17,12 +20,19 @@ function NavBar() {
         </Link>
 
       <div>
-        <button
+          {state && <button
```

Comment:
Je kan hier ook de ternary operator gebruiken. Jouw manier is ook helemaal correct, dus vooral doen wat jij fijn vindt werken.

```suggestion
          {state ? <button
              type="button"
              onClick={() => Toggle()}
          >
              Sign out
          </button>
          :
        <button
          type="button"
          onClick={() => navigate('/signin')}
        >
          Log in
        </button>
        }
```